descwl/analysis.py

```diff
@@ -67,7 +80,7 @@ def __init__(self,survey,table,stamps,bounds,num_slices):
         self.bounds = bounds
         self.num_slices = num_slices
         if len(self.stamps) > 0:
-             #3rd case allows second partials
+             #21 allows second partials
```

Comment:
Why 21?

Reply:
21 is the number of first partials + second partials after excluding the flux and taking into account that partial derivatives commute, i.e.
1 +  (6-1) + (6 * 6 - 6)/2

The first term corresponds to the galaxy image itself , the second term is the first partials excluding the flux one (just normalized image), and the last term is the number of second partials excluding the flux ones and taking into account commutation.

Reply:
@fjaviersanchez , should I make an explicit comment on the code about this?

Reply:
comment added